### PR TITLE
feat(route): explicitly fall back to the next handler

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: docs/src/api/class-route.md [hunk 1]
```diff
@@ -114,6 +114,120 @@ If set changes the post data of request
...
+  }
+  // Handling POST only.
+  // ...
+});
+```
+
+```python async
+# Handle GET requests.
+def handle_post(route):
+    if route.request.method != 'GET':
```
Comment (dgozman) [inline docs/src/api/class-route.md:170]:
```suggestion
    if route.request.method != "GET":
```

File: docs/src/api/class-route.md [hunk 2]
```diff
@@ -114,6 +114,120 @@ If set changes the post data of request
...
+        route.fallback()
+        return
+  # Handling GET only.
+  # ...
+
+# Handle POST requests.
+def handle_post(route):
+    if route.request.method != 'POST':
```
Comment (dgozman) [inline docs/src/api/class-route.md:178]:
```suggestion
    if route.request.method != "POST":
```

File: docs/src/api/class-route.md [hunk 3]
```diff
@@ -114,6 +114,120 @@ If set changes the post data of request
...
+  # ...
+
+await page.route("**/*", handle_get)
+await page.route("**/*", handle_post)
+```
+
+```python sync
+# Handle GET requests.
+def handle_post(route):
+    if route.request.method != 'GET':
```
Comment (dgozman) [inline docs/src/api/class-route.md:191]:
```suggestion
    if route.request.method != "GET":
```

File: docs/src/api/class-route.md [hunk 4]
```diff
@@ -114,6 +114,120 @@ If set changes the post data of request
...
+        route.fallback()
+        return
+  # Handling GET only.
+  # ...
+
+# Handle POST requests.
+def handle_post(route):
+    if route.request.method != 'POST':
```
Comment (dgozman) [inline docs/src/api/class-route.md:199]:
```suggestion
    if route.request.method != "POST":
```

File: docs/src/api/class-route.md [hunk 5]
```diff
@@ -114,6 +114,120 @@ If set changes the post data of request
...
+
+page.route("**/*", handle_get)
+page.route("**/*", handle_post)
+```
+
+```csharp
+// Handle GET requests.
+await page.RouteAsync("**/*", route => {
+    if (route.Request.Method != "GET") {
+        route.Fallback();
```
Comment (dgozman) [inline docs/src/api/class-route.md:213]:
```suggestion
        await route.FallbackAsync();
```

File: docs/src/api/class-route.md [hunk 6]
```diff
@@ -114,6 +114,120 @@ If set changes the post data of request
...
+        return;
+    }
+    // Handling GET only.
+    // ...
+});
+
+// Handle POST requests.
+await page.RouteAsync("**/*", route => {
+    if (route.Request.Method != "POST") {
+        route.Fallback();
```
Comment (dgozman) [inline docs/src/api/class-route.md:223]:
```suggestion
        await route.FallbackAsync();
```

**Review by dgozman — APPROVED** [review]

File: tests/library/browsercontext-route.spec.ts [hunk 7]
```diff
@@ -250,19 +250,19 @@ it('should overwrite post body with empty string', async ({ context, server, pag
...
 
-it('should chain continue', async ({ context, page, server }) => {
+it('should chain fallback', async ({ context, page, server }) => {
   const intercepted = [];
   await context.route('**/empty.html', route => {
     intercepted.push(1);
-    route.continue();
+    route.fallback();
   });
   await context.route('**/empty.html', route => {
```
Comment (rwoll) [inline tests/library/browsercontext-route.spec.ts:259]: Should we add a test to make sure `fallback` works with the new [async handlers](https://github.com/microsoft/playwright/pull/14317)?

**Review by rwoll — APPROVED** [review]
LGTM, modulo adding async test assuming it's supported.

File: tests/library/browsercontext-route.spec.ts (hunk 7 above)
Comment (pavelfeldman) [reply tests/library/browsercontext-route.spec.ts:259]: Should be covered!

**Pull request merged**